git-hg/README.md

```diff
@@ -4,4 +4,40 @@ These scripts are run at https://ci.adoptopenjdk.net/view/git-hg/ and are respon
 
 ## For developers
 
-TBA
+OpenJDK Source Control repositories are managed using Mercurial. The scripts within this project mirror those Mercurial repositories
+to github.com/AdoptOpenJDK/openjdk-jdk\<NN\>u , which then serves as a mirror for AdoptOpenJDK to build the Hotspot variants, as well as providing branches for adding any AdoptOpenJDK vendor "patches" needed.
+
+The OpenJDK Mercurial repositories fall under three categories:
+- JDK8 Update Repositories : hg.openjdk.java.net/jdk8u/jdk8u/(corba|hotspot|jaxp|jaxws|nashorn|jdk|langtools)
```

Comment:
probably look a bit better in code blocks

Reply:
done